tests/charm++/zerocopy/large_p2p/Makefile

```diff
@@ -0,0 +1,27 @@
+-include ../../../common.mk
+-include ../../../../include/conv-mach-opt.mak
+CHARMC=../../../../bin/charmc $(OPTS)
+
+all: large_p2p
+
+OBJS = large_p2p.o
+
+large_p2p: $(OBJS)
+	$(CHARMC) -language charm++ -o large_p2p $(OBJS)
+
+cifiles: large_p2p.ci
+	$(CHARMC)  large_p2p.ci
+	touch cifiles
+
+large_p2p.o: large_p2p.C cifiles
+	$(CHARMC) -c large_p2p.C
+
+test: all
+	$(call run, +p1 ./large_p2p 2147479558 )
+	$(call run, +p2 ./large_p2p 2147479558 )
+ifeq ($(CMK_USE_CMA),1)
+	$(call run, +p2 ./large_p2p 2147479558 +noCMAForZC )
+endif
```

Comment:
Does this test succeed on i386?

Reply:
Could also add a comment saying that the parameter passed in is `MAX_CMA_RW_COUNT + 6`.

Reply:
Yes, it ran fine on `netlrts-linux-i386` build. Is that what you were asking?

Reply:
Added the comment.

Reply:
> Yes, it ran fine on `netlrts-linux-i386` build. Is that what you were asking?

Yes.